client/src/index.css

```diff
@@ -1,4 +1,8 @@
-
+body {
```

Comment:
I'm not sure if we should keep the CSS I wrote at the start of project ?
Do you need it ?

Reply:
you can keep it